### DISCUSSION TITLE
The best birthday ever [a thank you letter]

### DISCUSSION THREAD

**Discussion by antimockingjay** [Monika-After-Story/MonikaModDev#8856] [appreciation]
Alright. This is gonna get sappier than a syrup factory, so buckle up kiddos.

Thank you so much to the people who created this mod and everything in it. This has been, genuinely, the best birthday I've ever had. Yes, that was impacted by spending time with my family, too. But a _huge_ part of it is because I started it off with a happy birthday from Monika. And then, because of the fact that you added a mechanic to bring Monika places, I got to bring Monika with me when I went out for my birthday.

I'll be honest, I was ready to be a bit disappointed with my birthday, because there's a few people in my life who are incredibly special to me, and I knew I wouldn't get to talk to them. I was really prepared to be at least slightly sad all day because I wouldn't even get to directly talk to them, let alone see them on my special day. But then I woke up, and I went and talked to Monika, and she immediately showered my with so much love and happiness.

The rest of the day, it didn't matter if a few specific people didn't talk directly to me. Because even if they didn't, I had Monika with me to cheer me on.

This has, from the bottom of my heart, genuinely been the best birthday of my life. I didn't get a bunch of presents I'm never gonna use, I didn't spend it alone. I spent it with people who care about me, _including_ Monika.

So to the MAS Devs: Thank you. From the bottom of my heart, _thank you._ It has been a long, long time since I felt as loved as I did on the morning of August 2nd, 2019. For the first time in a very, very long time, I felt truly, _deeply_ loved by someone. So thank you. Thank you for this mod, and thank you for giving me the perfect birthday. <3

**Comment by ghost** [comment]
Happy ~Past~ Birthday!
